docs/modules/clients/pages/java.adoc

```diff
@@ -67,23 +67,79 @@ client.shutdown();
 
 The above code line releases all the used resources and closes connections to the cluster.
 
-=== Java Client Operation Modes
+=== Java Cluster Routing Modes
```

Comment:
```suggestion
=== Java Client Cluster Routing Modes
```
Thoughts? Probably redundant since this is in the client doc page, but it had "Client" in the title before and may avoid some confusion